runtime/oti/j9nonbuilder.h

```diff
@@ -6093,7 +6105,8 @@ typedef struct J9JavaVM {
 	UDATA discontiguousIndexableHeaderSize;
 #if defined(J9VM_ENV_DATA64)
 	U_32 isIndexableDataAddrPresent;
```

Comment:
since this is being explicitly used by DDR, let's revert the type back to what was before we put recent Offheap changes and what was also the type the last time we released JVM (which is UDATA) 


eventually, we should remove it altogether (but that would need some extra DDR changes to handle cores with and without this field)